src/error.rs

```diff
@@ -47,8 +47,6 @@ impl Error {
     /// Return the error code associated with this error.
     pub fn code(&self) -> ErrorCode {
         match self.raw_code() {
-            raw::GIT_OK => super::ErrorCode::GenericError,
-            raw::GIT_ERROR => super::ErrorCode::GenericError,
```

Comment:
Could this (and other deletions from this file in this commit) stay? They're intended to be listed in the matches.